Add Anchor connector

mre opened the pull request:
First draft of Anchor connector. Not tested yet as API endpoints need to be added first.

woolfg commented on .github/workflows/docker-anchor.yml:
should we deprecate this method of calling/building a docker image and integrate it using the DB cred approach. not sure if it makes sense to maintain two approaches to run it.

mre replied:
I've added it for debugging, mostly. Let's add it and then remove all the Docker pipelines in one go in a bit. Makes more sense from a "changeset" perspective.

woolfg reviewed (commented):
- I would suggest to deprecate the single fetcher approach and combine all fetchers using the connector_manager approach. this would have the advantage of merging the helper modules without copying it and maintaining just one approach and code base which can share all helper function.
- the integration in the connector_manager is missing

mre commented:
* [x] Yes, let's deprecate the single fetcher approach and combine all fetchers using the connector_manager approach.
We can do that as a next step. For now, I've added the files for the single fetcher for completeness' sake.
* [x] Added the integration in the connector_manager.

Tested locally and imports all Anchor data for our test podcast. (No API Endpoints yet.)
This is ready for another round of review now.
